src/permutat.c

```diff
@@ -4753,7 +4726,11 @@ static Int InitKernel (
     InfoBags[           T_PERM4         ].name = "permutation (large)";
     InitMarkFuncBags(   T_PERM4         , MarkNoSubBags );
 
-    /* install the type functions                                          */
+    MakeBagTypePublic( T_PERM2);
+    MakeBagTypePublic( T_PERM4);
```

Comment:
I think these two calls should be in #ifdef HPCGAP or whatever, or MakeTypeBagPublic should be renamed,